services/src/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/switchmanager/SwitchManager.java

```diff
@@ -668,6 +670,66 @@ public long installOneSwitchFlow(DatapathId dpid, String flowId, Long cookie, in
         return flows;
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public List<MeterEntry> getExpectedDefaultMeters(DatapathId dpid, boolean multiTable, boolean switchLldp)
+            throws SwitchOperationException {
+        List<MeterEntry> meters = new ArrayList<>();
+        IOFSwitch sw = lookupSwitch(dpid);
+        boolean packetsFlag = featureDetectorService.detectSwitch(sw).contains(SwitchFeature.PKTPS_FLAG);
+
+        meters.add(getExpectedDefaultMeter(sw, VERIFICATION_BROADCAST_RULE_COOKIE, true, packetsFlag));
```

Comment:
Can we use the same code here that we use to install meters on the switch? (maybe it’s worth a little rewrite of the code if we cannot use it now). I would like to avoid copy-paste and be always sure that we are checking exactly what we are putting on the switch.

Reply:
Reworked

Reply:
Reworked.